features/_your_results/limit_tax_relief.feature

```diff
@@ -0,0 +1,68 @@
+Feature:
+  As an employee contributing greater than £40,000 per year to my pension
+  In order to get a clear picture of the impact on my take-home pay.
+  I want the results to reflect the fact that I will not get tax relief
+  for any contribution over £40,000 per year.
+
+  Background:
+    Given I am on the Your Details step
+
+    Scenario: Limit tax relief when above yearly figures
+      Given I fill in my details:
+        | age | gender | salary | salary_frequency | contribution |
+        | 25  | male   | 80000  | per Year         | Full         |
+      And I click the Next button
+      And I fill in my contributions:
+        | your_contribution | employer_contribution |
+        | 60                | 1                     |
+      When I move on to the results page
+      Then I should see on the results page:
+        | £48,000.00 | £48,000.00 | £48,000.00 |
+        | £8,000.00  | £8,000.00  | £8,000.00  |
+        | £800.00    | £1,600.00  | £2,400.00  |
+        | £48,800.00 | £49,600.00 | £50,400.00 |
+
+    Scenario: Limit tax relief when above weekly figures
+      Given I fill in my details:
+        | age | gender | salary | salary_frequency | contribution |
```

Comment:
if `age`, `gender` and `contribution` are incidental details then I think we could remove these out of the steps..?
you could even use the step `And I fill in my details` on the Background. and then have the step 
`Given my salary is '£80000, per year'` and pass `80000` and `per year` as arguments